reactioncommerce/reaction
# Show proper error messages when Stripe payments fail

Resolves #2483 

Sanitize error messages from server and pass them directly up to the client

### Testing

1. Configure Stripe with an invalid API key
1. Attempt to check out
1. Observe the error message contains no sensitive information

1. Configure Stripe with a valid API key
1. Attempt to check out using `4000000000000002` as your CC so that your card will be declined
1. Observer that a "Your card was declined" is displayed to the client
1. Attempt to check out with a card number of `4242424242424141` (so that Luhn check fails)
1. Observe you get an appropriate error
